EpikCord/__init__.py

```diff
@@ -1924,7 +1946,7 @@ async def connect(self, guild_id:str, channel_id:str, **kwargs):
         self.ip:str = ready_parsed_data["ip"]
         self.port:int = ready_parsed_data["port"]
         self.encryption_modes:list = ready_parsed_data["modes"]
```

Comment:
If you are trying to typehint on this, do `"list[str]"` or if there is typing module imported, `typing.List[str]`

Reply:
yeah